actions/docker-publish/README.md

```diff
@@ -0,0 +1,35 @@
+# docker-publish
+
+This action downloads an `image.tar` file from a pipeline artifact and publishes it on Dockerhub.
+
+## Prerequisites
+
+Create an action that uploads an `image.tar` file as pipeline artifact (see <https://github.com/actions/upload-artifact>).
```

Comment:
The action is already created by you! 😄 Just point out to our own GitHub action (`java-gradle-build-jib-image`), which actually does that!

Reply:
I kept this more generic, because you could use this with any type of action that builds a docker image, e.g. python-build-docker or docker-build for Dockerfiles, but we can still link this action as an example